xray/conventions.py

```diff
@@ -130,7 +133,7 @@ def decode_cf_datetime(num_dates, units, calendar=None):
         dates = (pd.to_timedelta(num_dates.ravel(), delta) + ref_date).values
```

Comment:
@shoyer : I was just looking through this code trying to understand an error I was getting when I noticed that a bit of unnecessary redundancy here.  On line 120, `flat_num_dates = num_dates.ravel()`.  `flat_num_dates` is used in  `_decode_netcdf_datetime` but not here where `num_dates.ravel()` is computed again.  Since it is possible that `ravel()` makes a copy, it would be better to choose one or the other.